classes/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Kindly pass in the other parameters for creating a person. We need to pass in the name, age and parent_permission.

- [ ] We also need to follow the order of the parameters in the `#super` method.

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
  ```